Backend/app/routes/notification.py

```
@@ -0,0 +1,208 @@
+from fastapi import APIRouter, Depends, HTTPException, status, Body, Request, Header
```

Ruff (0.11.9)

`fastapi.Request` imported but unused

Remove unused import: `fastapi.Request`

(F401)